TUTORIAL.md

```diff
@@ -0,0 +1,192 @@
+
+### No abbreviations
+
+Abbreviations are another source of potential confusion for both native and non-native English speakers. As a result, most uses of abbreviations should be avoided. The only acceptable abbreviation is the shortening of "et cetera" to "etc." since the abbreviated form is more common and, therefore, more likely to be recognized by learners.
+
+### Acronyms are expanded on first use
+
+Acronyms are acceptable for use in tutorials, but they should almost always be expanded on first use. For example, if a tutorial mentions the "DOM", then it should be written as "DOM (Document Object Model)" the first time it is used. After the first instance, acronyms should be written on their own. This rule also applies to shortened forms of long terms such as internationalization. This word can be written as "i18n (internationalization)" when first introduced and then simply as "i18n" thereafter.
+
+### Spell-checked against US-English dictionary
+
+Dojo 2 is being created and maintained by a multi-cultural community and, as a result, there are several versions of "English" that are represented. All tutorials should be spell checked against a standard US English dictionary.
+
+### 'aside' tags used for short, supporting script
+
+In the course of teaching a concept, there are often points that are useful for the learner to be made aware of but don't fit within the tutorial itself. In these situations, consider using the `aside` tag to allow the desired content to be added without disrupting the overall flow of the tutorial. In general, asides should be fairly short (one or two sentences) and placed as close to the relevant content.
+
+Example:
+
+```markdown
+{% aside "The aside's title" %}
+  Put the aside's content here. No more than a sentence or two, please!
+{% endaside %}
+
+```
+
+Note: The `aside` tag can render markdown for hyperlinks as well as inline code samples, but the results should always be checked before publication.
+
+### 'include_codefile' tags used for code-snippets, where possible
+
+Since Dojo 2 tutorials are demo-driven, they tend to contain a lot of code snippets. Keeping the code in the demo projects synchronized with the tutorials can be simplified by using the `include_codefile` tag in the text. There are several ways to configure the `include_codefile` tag to point include a code snippet.
+
+- The simplest form is to simply provide a relative path from the current file to the file that should be included:
+
+ ```markdown
+
+{% include_codefile 'demo/finished/biz-e-corp/src/widgets/HelloWorld.ts' %}
+
+```
+
+This will replace the tag with the file's contents and apply syntax-highlight to it with the assumption that the contents are JavaScript code.
```

Comment:
It will use JavaScript rules for now. There seems to be something wrong with using TypeScript. I've created issue #121 to address it.